docs/docker/docker-images.md

```diff
@@ -0,0 +1,218 @@
+# Docker Images
+
+## Building your own image
+Please refer [Build your own image](https://docs.docker.com/mac/step_four/) for more reference.
+
+The below steps only provide instructions for creating and building docker images.
+
+### Build image using docker file
+The `docker build -t ng2-book-finder .` command takes the `Dockerfile` in the current directory, and builds an image called `ng2-book-finder` on your local machine.
+
+First Docker checks to make sure it has everything it needs to build.
+
+```
+ng2-book-finder (feature/docker-image) $ docker build -t ng2-book-finder .
+Sending build context to Docker daemon   214 kB
+```
+
+Then, Docker loads with the `node:4.2-slim` image, as the image is not present on the current docker-machine, it downloads the image.
+
+```
+Step 1 : FROM node:4.2-slim
+4.2-slim: Pulling from library/node
+Status: Downloaded newer image for node:4.2-slim
+ ---> 264411d7f3f9
+```
+
+The below steps are performed for building the image.
+ - Expose the port
+ - set the required working directory
+ - copy required files for the image.
+ - Do `npm install`.
+ - And then compile the typescript to javascript.
```

Comment:
Next time ,  just take the screen shot of this log  as image and show it ..  that would differentiate log statement/ console statement from doc